frontend/www/js/omegaup/components/admin/Roles.vue

```diff
@@ -45,20 +45,19 @@ export default class AdminRoles extends Vue {
   groups: omegaup.Group[] = this.initialGroups;
 
   @Emit()
-  onChangeRole(ev: Event, role: string): omegaup.Role {
+  onChangeRole(ev: Event, role: string): omegaup.Selectable<string> {
```

Comment:
```suggestion
  onChangeRole(ev: Event, role: string): omegaup.Selectable<Role> {
```

igual en los demás lugares, hay que usar el tipo correcto.

Reply:
Listo